pandas/core/internals/blocks.py

```diff
@@ -2403,7 +2404,8 @@ def fillna(self, value, **kwargs):
     def to_native_types(self, na_rep="NaT", **kwargs):
```

Comment:
can you now type all of these to return a Block?

Reply:
sure

Reply:
actually adding this makes mypy complain about signature mismatch, rather not futz with it ATM